bundles/automation/org.eclipse.smarthome.automation.module.script.rulesupport/src/main/java/org/eclipse/smarthome/automation/module/script/loader/internal/file/ScriptFileProvider.java

```diff
@@ -0,0 +1,217 @@
+/**
```

Comment:
yes it is used. I moved it to the rulesupport bundle because you told me so. We previously had the discussion about the loaders and general usage of the script support. Then you told me to move it to rulesupport, yes I think it does not belong there, but I did not want to argue anymore for the first merge, so I moved it. You also told me, I do not have to rename the namespaces ;)